src/pathNotEq.js

```diff
@@ -0,0 +1,29 @@
+import { pathEq, complement } from 'ramda';
+
+
+/**
+ * Determines whether a nested path on an object has not a specific value,
+ * in R.equals terms. Most likely used to filter a list.
+ *
+ * @func pathNotEq
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Relation
+ * @sig  [k] → a → {a} → Boolean
+ * @param {Array} path The property paths to fetch
+ * @param {a} val The value to compare to
+ * @param {Object} object The object, that presumably contains value under the path
+ * @return {Boolean} Comparison result
```

Comment:
Returns Boolean `false` if the value equals the nested object property, `true` otherwise